bot.py

```diff
@@ -112,13 +121,116 @@ def on_message(ws, message):
         if admin_message(response):
             return
 
-    elif "team_join" == response["type"]:
-        if NEWUSERGREETING:
-            new_user_message(response)
-        USER_LIST = sc.api_call(  # noqa: 841
-            "users.list",
-        )
+    log(response)   
 
+def log(response):
+    global engine
+    def add_user(r, team_join=False):
+        users_with_this_slack_id = engine.execute("SELECT last_name FROM users WHERE slack_id=%s", r["user"])
```

Comment:
You shouldn't use pure sql if you're using sqlalchemy. Heres how to do it programmatically: 

http://docs.sqlalchemy.org/en/latest/core/tutorial.html#selecting